container/image.bzl

```diff
@@ -744,6 +749,12 @@ _attrs = dicts.add(_layer.attrs, {
 
         This field supports stamp variables.""",
     ),
+    "override_tag": attr.string(
+        doc = """Override final tag name.
+
+        By default the final docker image name is bazel/{target}:{name}.
+        This flag overrides the string after the colon.""",
+    ),
```

Comment:
Can we just call it `tag` or `tag_name` and document that if not specified, it would be {name}?